test/performance/tools/common.sh

```diff
@@ -96,6 +96,7 @@ function update_cluster() {
   pushd .
   cd ${GOPATH}/src/knative.dev
   echo ">> Update istio"
+  kubectl delete all --all --ignore-not-found --now --timeout 60s -n istio-system
```

Comment:
While this cleans up most resources, there are still other cluster-scoped resources like webhook configurations and CRDs.

Please add a comment on why (if) those left-over resources don't matter.